lxd/main_callhook.go

```diff
@@ -49,6 +209,7 @@ func (c *cmdCallhook) Run(cmd *cobra.Command, args []string) error {
 	var projectName string
 	var instanceRef string
 	var hook string
+	var hooksFiles []string // Used for startmountns hook only.
```

Comment:
these are CDI specific, so CDI should be mentioned in this var i think

Reply:
fixed